### PR TITLE
Allow to request market cap and daily volumes for a set of coins.

### PR DESCRIPTION
**Pull request by ygavrishov**
* A wrapping method **GetAllPricesExtended** was added that takes not only prices but also market cap and daily volumes.
* Some existing integration tests were fixed.

### CODE REVIEW

**Comment by ygavrishov** [comment]
It's done to implement https://github.com/trakx/index-manager/issues/16

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # coingecko-api-client
 C# implementation of a CoinGecko API client
+
+## How to regenerate C# API clients
+
+* If you work with external API, you probably need to update Change OpenAPI definition added to the project. It's usually openApi3.yaml file.
```
Comment (monsieurleberre) [inline README.md:6]: automatic 🤩

**Review by monsieurleberre — APPROVED** [review]
thanks, another easy one to review! well done

**Comment by monsieurleberre** [comment]
I am merging to master so you can use a non temporary package.
Feel free to do PR to merge to master when you need that.

**Pull request merged**